packaging/debian/control

```diff
@@ -132,8 +132,8 @@ Suggests: ${sng:CoreModules}, ${sng:Modules}
 Provides: system-log-daemon, linux-kernel-log-daemon, libsyslog-ng-dev,
  syslog-ng-abi-${sng:ABI}
 Conflicts: system-log-daemon, linux-kernel-log-daemon
-Replaces: syslog-ng (<< 3.3.0~), libsyslog-ng-dev, syslog-ng-mod-json (<< 3.13.1~)
-Breaks: syslog-ng (<< 3.3.0~), libsyslog-ng-dev, syslog-ng-mod-json (<< 3.13.1~)
```

Comment:
I think the `Breaks`+`Replaces` is still needed, because the new core package has files that used to be elsewhere. The B/R pair help apt order the upgrade process so that it won't try to overwrite files from the old package and (rightfully) error out.